README.md

```diff
@@ -1,19 +1,241 @@
+_support branch_ (more on this later)
+* A beta release may be created at any time during the release cycle. These are used if early access is needed by
+external applications to an ONOS API change, or as a test of the release process. These are versioned like `2.5.7-b1`
+with ascending beta release numbers. Betas may be released from a support branch or `master`
+* A release candidate is created at the end of a release cycle, usually to give QA a chance to test the final versions
+of a release. These are usually built from support branches. They are versioned like `2.5.7-rc1` with ascending 
+release candidate numbers.
 
+## Support branches
+Support branches are created late in th release cycle to freeze development of a release and allow further work to
+continue on the `master` branch. Once a support branch is created, any bug fixes to that release must go on `master`
+as well as the support branch. Official releases are _always_ built from support branches. The release branch name is
+derived from the major and minor numbers of the release. So for example, the support branch for ONOS 2.5 is called 
+`onos-2.5`. All 2.5 point releases are built from this branch (`2.5.0`, `2.5.1`, etc.)
 
-ONOS Configuration subsystem built using the µONOS architecture
+### Procedure to create a release branch
+```shell
+# Start on master, with an up to date tree
+# set the release branch name here
+export RELEASE="2.5"
+export NEXT_RELEASE="2.6"
+git checkout master 
+git pull 
 
-You can find all the documentation under [docs](docs) directory. 
-A good place to start is the [README](docs/README.md). 
+# Create the new branch 
+git checkout -b onos-$RELEASE master 
 
-Overall µONOS documentation is also published on its own [website](https://docs.onosproject.org).
+# now on new branch 
+# edit .gitreview, set defaultbranch to new branch name 
+defaultbranch=onos-$RELEASE
+
+# push the new branch to gerrit 
+git push origin onos-$RELEASE
+
+# commit your change to .gitreview 
+# you will have to verify this by hand, there is no verify job for the new 
+# branch yet 
+git add .gitreview 
+git commit -m "bumping to version $RELEASE"
+git review 
+
+# go back to master 
+git checkout master 
+
+# create a new local branch for the next release on master
+git checkout -b branch-$NEXT_RELEASE
+
+# change the versions to be the next release. 
+onos-snapshot $NEXT_RELEASE.0-SNAPSHOT 
+
+# remove debris left behind by the version setting 
+find . -name *.versionsBackup | xargs rm 
+
+# put the version changes into gerrit 
+git review 
+
+# when the verification jobs finish, +2 and submit this change 
+
+# now add jobs to verify the new support branch 
+# check out the ci-management tree 
+# edit jjb/onos/onos-verify.yaml 
+# add the new branch to the onos and onos-gui verify jobs 
+# submit and merge these changes 
+```
+
+## ONOS Release Build
+
+The first step to a release is the release build. This will create the JAR files and other artifacts that comprise 
```

Comment:
from here on it reads like it only refers to ONOS classic - JARs, bazel, Karaf - I don't understand how this is relevant to onos-config

Reply:
Marked as "do not merge" because it was an experiment.